src/System.Security.Principal.Windows/src/System/Security/Principal/WindowsIdentity.cs

```diff
@@ -46,9 +55,47 @@ public class WindowsIdentity : ClaimsIdentity, IDisposable, ISerializable, IDese
 
         private static bool s_ignoreWindows8Properties;
 
-        //
-        // Constructors.
-        //
+        public WindowsIdentity(IntPtr userToken) : this(userToken, null, -1) { }
+
+        public WindowsIdentity(IntPtr userToken, string type) : this(userToken, type, -1) { }
+
+        public WindowsIdentity(IntPtr userToken, string type, WindowsAccountType acctType) : this(userToken, type, -1) { }
```

Comment:
We should add a comment that acctType is ignored and the actual account type will always be retrieved from the system.